migrations/capcons/storagecapmigration.go

```diff
@@ -73,14 +73,26 @@ func (m *StorageCapMigration) CanSkip(valueType interpreter.StaticType) bool {
 
 func IssueAccountCapabilities(
```

Comment:
I wonder if we should move / replicate this function in flow-go, so we have more and easier control there over how to handle cases like e.g. a missing borrow type. Once we decide to e.g. no longer skip, we'd have to update this logic here and tag yet another Cadence release. Not sure, just a side-note

Reply:
yeah had the same thought - it would definitely be easier. But then again, it's like any-other migration (cap-con, link migration, etc), the migration logic is in Cadence, and only the flow-go makes use of it.